conans/client/cache/cache.py

```diff
@@ -291,7 +290,9 @@ def reset_default_profile(self):
 
     def initialize_settings(self):
         if not os.path.exists(self.settings_path):
-            save(self.settings_path, normalize(get_default_settings_yml()))
+            settings_yml = default_settings_yml
```

Comment:
Could we make the same for `get_default_client_conf` to simplify the `normalize(get_default_client_conf()`?

Reply:
Not worth it, the ``conan.conf`` is going to dissapear, be replaced by new ``global.conf`` and profiles conf. It will eventually be completely removed when the remaining configs in it are migrated to the new stuff.